commands/config.ts

```diff
@@ -10,8 +11,6 @@ exports.describe = i18n(`${i18nKey}.describe`);
 
 exports.builder = yargs => {
   addConfigOptions(yargs);
```

Comment:
The issue said to remove all the flags but removing the `--config` flag breaks the logic